src/ResourceManager/Insights/Commands.Insights/help/Add-AzureRmLogProfile.md

```diff
@@ -8,7 +8,9 @@ schema: 2.0.0
 # Add-AzureRmLogProfile
 
 ## SYNOPSIS
-Creates a log profile.
+Creates a new activity log profile, there can be only one log profile per subscription. This profile is used to either archive the activity log to a storage account or stream it to an Azure event hub. Only a single standard storage account (premium storage account is not supported) per subscription is supported. It could either be of type ARM or Classic. If it's logged to a storage account, the cost of storing the activity log is billed at normal standard storage rates. 
+
+In the activity log, events can pertain to a region or could be "Global". Global essentially means these events are region agnostics and are independent of region, in fact majority of events fall into this category. If the activity log profile is set from the portal, it implicitly adds "Global" along with any other region selected in the user interface. When using the cmdlet, the location as "Global" must be explicitly mentioned apart from any other region.  
```

Comment:
> "Global"

"global" should be lower case to match what people see.

This commend is very good. Thanks for adding it.